app/services/partner_reporter_service.rb

```diff
@@ -0,0 +1,23 @@
+module PartnerReporterService
```

Comment:
I like the approach of adding a module or class to handle generation of data.

@davidrouten -- this is similiar to the idea you had for https://github.com/rubyforgood/diaper/issues/2092 ?

Reply:
Yep! I added my service file under `/app/services/surveys` but the idea is the same.

I'm still not 100% sure how to handle the return of my data. on #2092, there are like 12 different calculations that need to be run and returned, so I'm not sure if it's going to be a simple Ruby object with keys + the data, or it'll be a key + an object with a header name--for CSV generation or for using to display on a report landing page--and the data, or something similar.